forms-flow-web/src/components/Modals/Modal.scss

```diff
@@ -0,0 +1,146 @@
+
+.modal-header {
+  justify-content: space-between !important;
+}
+.modal-footer {
+    padding-top: 32px !important;
+    padding-bottom: 32px !important;
+}
+
+.modal-dialog {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-height: auto; /* Full height of the viewport */
+  margin: 0; /* Remove default margin */
+}  
+
+.modal-content {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    width: 100% !important;
+    color: var(--ff-modal-color);
+    pointer-events: auto;
+    background-color: var(--ff-modal-bg);
+    background-clip: padding-box; 
+    border:none;
+    border-radius: var(--ff-modal-border-radius);
+    outline: 0;
+}
+.modal.fade .modal-dialog {
+  transition: transform .7s ease-out, opacity 0.6s ease-out !important;
+  transform: translateY(-20%) !important;
+  opacity: 0; 
+}
+
+.modal.show .modal-dialog {
+  transform: translateY(0) !important; 
+  opacity: 1; 
+}
+
+.create-form-modal-body {
+    padding: 0 !important;
+}
+
+.modal-60w {
+    width: 60% !important;
+    max-width: 60% !important;
+}
+.modal-100W {
+    width: 60% !important;
+}
+.modal-50w {
+    width: 50% !important;
+    max-width: 50% !important;
+}
+
+.modal-header {
+    margin-left: 26px;
+    padding-top: 32px;
+    padding-bottom: 32px;
+    padding-right: 40px !important;
+    flex-shrink: 0;
+    color: var(--gray-gray-80, #303436);
+    font-family: Roboto;
+    font-size: 24px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: normal;
+}
+
+.modal-content {
+    border-radius: 16px;
+}
+
+.form-modal-content {
+    width: 900px !important;
+    height: 255px !important;
+    flex-shrink: 0 !important;
+    border-radius: var(--Rem1_5, 24px) !important;
+    position: absolute !important;
+    border: none !important;
+}
+
+.inner-div {
```

Comment:
Please change the naming to context specific. 
Instead of inner-div , you can use 'inner-form-content' . Avoid using html tag names like div,span as class names